README.md

```diff
@@ -78,6 +78,22 @@ print(tokenizer.decode(outputs[0], skip_special_tokens=True))
 # prints something like: Preheat the oven to 350 degrees and place the cookie dough in a baking dish [...]
 ```
 
+> [!NOTE]
```

Comment:
IMO this README entry can be removed, as we generally don't highlight individual models here.